examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -30,11 +30,13 @@
 .. footbibliography::
```

Comment:
move references to the bottom of the file

Reply:
Done.